### PR TITLE
News updates and man page fixes

### PR DESCRIPTION
**Pull request by daviesrob**
Updated NEWS for samtools.  Hopefully nothing important has been left out.

Includes some man page updates - `sam.gz` output and indexing; and the `samtools fasta` default value for the `-F` option.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,79 +1,275 @@
...
+
+   Examples of problems that will now be rejected include @SQ lines with
+   no SN: tag, and @RG or @PG lines with no ID: tag.
+
+ * samtools sub-commands will now add '@PG' header lines to output sam/bam/cram
+   files.  To disable this, use the '--no-PG' option. (#1087; #1097)
+
+ * samtools now supports alignment records with reference positions greater
+   than 2 gigabases.  This allows samtools to process alignments for
+   species which large chromosomes like axolotl and lungfish.  Note that
```
Comment (jmarshall) [inline NEWS]:
```suggestion
   species which have large chromosomes, like axolotl and lungfish.  Note that
```

File: NEWS [hunk 2]
```diff
@@ -1,79 +1,275 @@
...
+
+ * samtools flags, flagstats, idxstats and stats now have aliases
+   flag, flagstat, idxstat and stat. (#934)
+
+ * A new global '--write-index' option has been added.  This allows output
+   sam.gz/bam/cram files to be indexed while they are being written out.
+   This should work with addreplacerg, depad, markdup, merge, split,
+   and view. (#1062)
+
+ * A global '--vebosity' option has been added to enable/disable
```
Comment (jmarshall) [inline NEWS]:
```suggestion
 * A global '--verbosity' option has been added to enable/disable
```

File: NEWS [hunk 3]
```diff
@@ -1,79 +1,275 @@
...
+
+ * HTSlib (and therefore SAMtools) now uses version 4 signatures by default
+   for its s3:// plug-in.  It can also write to S3 buckets, as long as
+   version 4 signatures are in use.  See HTSlib's NEWS file and
+   htslib-s3-plugin manual page for more information.
+
+ * HTSlib (and therefore SAMtools) no longer considers a zero-length
+   file to be a valid SAM file.  This has been changed so that pipelines such
+   as `somecmd | samtools ...` with `somecmd` aborting before outputting
+   anything will not propagate the error to the second command.
```
Comment (jmarshall) [inline NEWS]: Same typo copied over from the other one! :smile:

```suggestion
   anything will propagate the error to the second command.
```

File: NEWS [hunk 4]
```diff
@@ -1,79 +1,275 @@
...
+
+ * The meaning of decode_md, store_md and store_nm in the fmt-option section
+   of the samtools.1 man page has been clarified. (#898, thanks to Evan Benn)
+
+ * Fixed numerous memory leaks. (#892)
+
+ * Fixed incorrect macro definition on Windows. (#950)
+
+ * bedcov, phase, misc/ace2sam and misc/wgsim now check for failure to open
+   files.  (#1013, thanks to John Marshall)
```
Comment (jmarshall) [inline NEWS]: Also thanks to the person who reported it

```suggestion
   files.  (#1013, thanks to Julie Blommaert and John Marshall)
```

File: NEWS [hunk 5]
```diff
@@ -1,79 +1,275 @@
...
+     aligned base of a read. (#847; #842 reported by Tiffany Delhomme.
+     See also htslib issue #59 and pull request #699).
+
+   - In samtools mpileup, a deletion or pad on the reverse strand is now
+     marked with a different character ('#') than the one used on a forward
+     strand ('*'), if the '--reverse-del' option is used. (#1070)
+
+   - New option '--output-extra' can be used to add columns for user
+     selected alignment fields or aux tags. (#1073; #1100 thanks to
+     John Marshall)
```
Comment (jmarshall) [inline NEWS]: Just `(#1073, #1100)`; my contribution here was minimal.

**Comment by daviesrob** [comment]
Updated, along with removal of some stray trailing white-space.  I removed the YAML bit from markdup stats output as we (myself and @whitwham) weren't entirely sure that it was true.

**Pull request merged**